QModManager/QModManager.csproj

```diff
@@ -104,12 +104,17 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
     <Reference Include="UnityEngine">
+      <SpecificVersion>False</SpecificVersion>
       <HintPath>..\Dependencies\UnityEngine.dll</HintPath>
```

Comment:
Do we need both `UnityEngine.dll` and the other unity files referenced?

Reply:
Actually yes. `Monobehavior` is still defined within the new "UnityEngine.dll".  
Project won't compile without it.